ansible/environments/local/hosts.j2.ini

```diff
@@ -8,9 +8,9 @@ ansible ansible_connection=local
 
 [controllers]
 controller0         ansible_host=172.17.0.1 ansible_connection=local
-;{% if mode is defined and 'HA' in mode %}
-;controller1         ansible_host=172.17.0.1 ansible_connection=local
-;{% endif %}
+{% if mode is defined and 'HA' in mode %}
+controller1         ansible_host=172.17.0.1 ansible_connection=local
```

Comment:
This is to enable the second controller in the CI environment to run [the HA test for controllers](https://github.com/apache/openwhisk/blob/master/tests/src/test/scala/ha/ShootComponentsTests.scala#L145).